test-runner-poc/environments.js

```diff
@@ -43,15 +45,15 @@ function usingUnifiedTopology() {
 class ReplicaSetEnvironment extends EnvironmentBase {
   constructor(status, version) {
     super(status);
-    this.setName = status.setName || 'replset';
+    this.setName = status.setName || 'rs';
     this.url = `mongodb://${this.host}:${this.port}/integration_tests?rs_name=${this.setName}`;
     this.writeConcernMax = { w: 'majority', wtimeout: 30000 };
-    this.replicasetName = 'rs';
+    this.replicasetName = status.setName || 'rs';
```

Comment:
Let's just use `this.setName` here since `this.setName = status.setName || 'rs'`

Reply:
fixed

Reply:
Looks like this wasn't fixed